pages/portfolio.html

```diff
@@ -4,9 +4,41 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="/Users/miay/Developer/unit2-projects/personal-portfolio-site/styles/portfolio.css" rel="stylesheet">
   <title>Document</title>
 </head>
 <body>
-  
+  <header>
+    <h1>Projects</h1>
+  </header>
+  <section>
+    <h2>Task List</h2>
+    <li>backend project creating endpoints for creating and altering tasks</li>
+    <div>
+      <img id="python" alt="python" src="/Users/miay/Developer/unit2-projects/personal-portfolio-site/assets/python-logo.jpg">
```

Comment:
Make sure to use relative paths for all this images. Each needs to move up a folder to be able to get to the `assets` folder, so this would be `../assets/python-logo.jpg`